CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### Breaking changes
```

Comment:
This shouldn't be a different section see https://github.com/maplibre/maplibre-gl-js/blob/77b2782ad11cf6aba90f366a11462f4ac62230b8/CHANGELOG.md?plain=1#L9

Reply:
Updated.
I based it on the format used earlier in the file I modified
https://github.com/maplibre/maplibre-style/blob/16fc6a7eb84ea5a3c963474da09b3477e666ceb4/CHANGELOG.md?plain=1#L229-L241
If it shouldn't be a separate section maybe it would be a good idea to unify the changelogs entries written for previous versions?

Reply:
Sure, feel free to do so.